src/OrchardCore.Modules/OrchardCore.Workflows/Views/WorkflowType/Index.cshtml

```diff
@@ -14,34 +14,36 @@
     <input type="submit" name="submit.BulkAction" class="visually-hidden" />
     <input asp-for="Options.BulkAction" type="hidden" />
 
-    <div class="card mb-3 position-sticky action-bar">
-        <div class="card-body bg-primary p-3">
-            <div class="mb-0 has-search">
-                <div class="input-group input-group-sm w-sm-50 w-25 d-inline-flex has-search">
-                    <i class="fa fa-search form-control-feedback" aria-hidden="true"></i>
-                    <input asp-for="Options.Search" class="form-control" placeholder="@T["Search"]" type="search" autofocus />
+    <div class="card bg-primary mb-3 position-sticky action-bar">
+        <div class="card-body">
+            <div class="row gx-3">
+                <div class="col">
+                    <div class="has-search">
+                        <i class="fa fa-search form-control-feedback" aria-hidden="true"></i>
+                        <input asp-for="Options.Search" class="form-control" placeholder="@T["Search"]" type="search" autofocus />
+                    </div>
                 </div>
-                <div class="btn-group float-end">
-                    <a asp-action="EditProperties" asp-route-returnUrl="@FullRequestPath" class="btn btn-sm btn-secondary float-end create" role="button">@T["Create Workflow"]</a>
+                <div class="col-auto">
+                    <a asp-action="EditProperties" asp-route-returnUrl="@FullRequestPath" class="btn btn-secondary create" role="button">@T["Create Workflow"]</a>
                 </div>
             </div>
         </div>
     </div>
     <ul class="list-group with-checkbox">
-        @if (Model.WorkflowTypes.Any())
+        @if (Model.WorkflowTypes.Count > 0)
```

Comment:
I did not know that `Count` was better than `.Any()`.
We often use Any instead of Count in a lot of places.
Is it Visual Studio that gave you this recommendation?

Reply:
no. `Count` is a property that gets updated internally and always has the correct count. checking it is very cheap. when you use `.Any()` then `GetEnumerator()` and `MoveNext()` would have to be evaluated.

Reply:
Yes when `Count` or `Length` is available as a property but `Any()` is okay when only `Count()` is available.